firstservice/src/main/java/com/services/firstservice/api/controller/UserController.java

```diff
@@ -0,0 +1,50 @@
+package com.services.firstservice.api.controller;
+
+import javax.validation.Valid;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.services.firstservice.api.dto.UserDto;
+import com.services.firstservice.api.transformer.UserDtoTransformer;
+import com.services.firstservice.db.entity.user.User;
+import com.services.firstservice.service.UserService;
+
+@RestController
+@RequestMapping("/user")
+public class UserController {
+
+    @Autowired
+    private UserService userService;
```

Comment:
It's not a mistake to use `@Autowired`, but it's better to use `RequiredArgsConstructor` along with a `final` field.